scene/resources/shader.cpp

```diff
@@ -136,9 +150,38 @@ void Shader::get_shader_uniform_list(List<PropertyInfo> *p_params, bool p_get_gr
 			if (pi.type == Variant::RID) {
 				pi.type = Variant::OBJECT;
 			}
+#ifdef TOOLS_ENABLED
+			if (Engine::get_singleton()->is_editor_hint()) {
+#ifdef MODULE_REGEX_ENABLED
+				const RegEx pattern("/\\*\\*([^*]|[\\r\\n]|(\\*+([^*/]|[\\r\\n])))*\\*+/\\s*uniform\\s+\\w+\\s+" + pi.name + "(?=[\\s:;=])");
```

Comment:
Is this taking into account that we can have `instance uniform`?
There is also `global uniform`, though this one would not show docs I guess?
But `instance uniform` does appear in the inspector, right? I think it should show docs.

Reply:
Yes, IMHO the problem is that instance uniform belongs to per instance, as it's `_get_property_list` is in `visual_instance_3d`, but I think the impl can be transferred there.